WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -13,6 +13,13 @@ extension UIImage {
         return Gridicon.iconOfType(.addOutline)
     }
 
+    /// Notice Icon
+    ///
+    static var noticeImage: UIImage {
```

Comment:
We've been adding a simple unit test for each icon in `IconsTests`. When I started that, it made sense for icons that where UIImage(named:), to make sure that we did not ship with images missing from the Assets catalog (been bit by that in previous lifes).  I am not sure of the value of adding a test for GridIcons as well, but we've been doing it 🤷‍♂

Reply:
You did well to report it. I added the test. ;)